dom.bs

```diff
@@ -2004,8 +2004,9 @@ steps:</p>
  <li><p>If the <i>open flag</i> is set and <var>shadow</var>'s <a for=ShadowRoot>mode</a> is
  <em>not</em> "<code>open</code>", then return null.</p></li>
 
- <li><p>Return the first <a>slot</a> in <var>shadow</var>'s <a>tree</a> whose <a for=slot>name</a>
- is <var>slotable</var>'s <a for=slotable>name</a>, if any, and null otherwise.</p></li>
+ <li><p>Return the first <a>slot</a> in <var>shadow</var>'s <a for=tree>descendants</a> whose
+ <a for=slot>name</a> is <var>slotable</var>'s <a for=slotable>name</a>, if any and in
+ <a>tree order</a>, and null otherwise.</p></li>
```

Comment:
'in tree order' feels to be a bit out of place. Could it be earlier in the sentence?